docs/extending/adding_reports.md

```diff
@@ -2,34 +2,52 @@
 
 # Adding reports
 
-Reports are views with listings of pages matching a specific query. They can also export these listings in spreadsheet format.
+Reports are views with listings of pages or any non-page model (such as snippets) matching a specific query. Reports can also export these listings in spreadsheet format.
 They are found in the _Reports_ submenu: by default, the _Locked pages_ report is provided, allowing an overview of locked pages on the site.
 
-It is possible to create your own custom reports in the Wagtail admin. Two base classes are provided:
-`wagtail.admin.views.reports.ReportView`, which provides basic listing and spreadsheet export functionality, and
-`wagtail.admin.views.reports.PageReportView`, which additionally provides a default set of fields suitable for page listings.
+It is possible to create your own custom reports in the Wagtail admin with two base classes provided:
+
+-  `wagtail.admin.views.reports.ReportView` - Provides the basic listing and spreadsheet export functionality without any pre-determined fields.
+-  `wagtail.admin.views.reports.PageReportView` - Extends the `ReportView` and provides a default set of fields suitable for page listings.
+
+## Reporting reference
+
+```{eval-rst}
+.. attribute:: template_name
+...
```

Comment:
When I put the `...` in the issue, I meant this as an example of 'include the content...'. three dots does not do anything in RST syntax.